### PR TITLE
[RFC] Do not indent calls with a single template literal argument

### LINKED ISSUE #786: odd output for template literals
given the following code:

```js
insertRule(`*, *:before, *:after {
  box-sizing: inherit;
}`)
```

I get the following output:

```js
insertRule(
  `*, *:before, *:after {
  box-sizing: inherit;
}`,
)
```

The formatting of the last two lines seems odd to me. I'm not sure why a comma is added or why the line with the closing bracket is on the same indentation as the closing parenthesis.

### PR DESCRIPTION
**Pull request by vjeux**
I'm not sure that this is a good idea. A current workaround is to remove the parenthesis.

Fixes #786

### CODE REVIEW

File: tests/strings/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -123,21 +123,17 @@ pipe.write(
...
-  \`a long string \${1 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3} with expr\`
-);
+foo(\`a long string \${1 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3} with expr\`);
 
 const x = \`a long string \${1 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + (function() {
     return 3;
   })() + 3 + 2 + 3 + 2 + 3} with expr\`;
 
-foo(
-  \`a long string \${1 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + (function() {
```
Comment (vjeux) [inline tests/strings/__snapshots__/jsfmt.spec.js.snap]: this was written as being indented, and now is all messed up. So this isn't ideal for this situation :(

**Comment by jlongster** [comment]
Hm, yeah this is hard call. I don't feel good merging this without thinking a little more and seeing how it affects more people. Thanks for the idea!

**Comment by vjeux** [comment]
I'll close this to clean up the queue, we should re-open if we want to do it.

**Comment by vjeux** [comment]
It keeps coming up, it seems like it's how people actually write their code.

**Pull request merged**